packages/itwinui-css/src/button/button-group.scss

```diff
@@ -72,7 +71,8 @@
 
     // Add rounded corners to first and last items
     &:first-child {
-      #{$childBorderSelector} {
+      #{$childBorderSelector},
+      .iui-input-flex-container::before {
```

Comment:
why not move `.iui-input-flex-container::before` into `$childBorderSelector`?

Reply:
Css was not building.

Reply:
builds for me

Reply:
Yup, added